lib/versioning/node/index.ts

```diff
@@ -27,8 +29,21 @@ function getNewValue({
 
 export { isValid };
 
+export function isStable(version: string): boolean {
+  if (npm.isStable(version)) {
+    const major = npm.getMajor(version);
+    const schedule = nodeSchedule[`v${major}`];
+    if (schedule?.lts) {
+      // TODO: use the exact release that started LTS
+      return DateTime.local() > DateTime.fromISO(schedule.lts);
```

Comment:
Check end date too?

Reply:
If the end date has passed it doesn't mean the release is *unstable*, it fits better as *deprecated*